### PR TITLE
DCS- 1365 WPIP-api capitalise prisoner names

### PR DESCRIPTION
**Pull request by GurnankCheema**
example output in postman. See Alishony and Nancer rather than alishony and nancer
Screenshot 2021-12-21 at 16 11 47

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/basm/BasmServiceTest.kt [hunk 1]
```diff
@@ -13,18 +13,6 @@ class BasmServiceTest {
   private val basmClient: BasmClient = mockk()
   private val service = BasmService(basmClient)
 
-  private val movement = Arrival(
```
Comment (GurnankCheema) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/basm/BasmServiceTest.kt:16]: not used anywhere

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/basm/BasmServiceTest.kt [hunk 2]
```diff
@@ -36,7 +24,7 @@ class BasmServiceTest {
       listOf(
         Arrival(
           id = "476d47a3-013a-4772-94c7-5d043b0d0574",
-          firstName = "Alexis",
+          firstName = "Alexis Robert",
```
Comment (GurnankCheema) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/basm/BasmServiceTest.kt:27]: expected output

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/basm/BasmTestData.kt [hunk 3]
```diff
@@ -51,7 +51,7 @@ class BasmTestData {
           id = "45a2b4a8-38ec-46f4-b882-148a21ebbe6e",
           person = People(
             id = "bd1bf67e-d160-4032-ad59-8f62cd7b25fe",
-            first_names = "Alexis",
+            first_names = "Alexis robert",
```
Comment (GurnankCheema) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/basm/BasmTestData.kt:54]: potential input with lower case first letter

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**